src/qtui/menus.cc

```diff
@@ -72,8 +72,13 @@ QMenuBar * qtui_build_menubar (QWidget * parent)
         audqt::MenuSep (),
         audqt::MenuCommand ({N_("Search _Library"), "edit-find", "Ctrl+Y"}, show_search_tool),
         audqt::MenuSep (),
+#ifdef Q_OS_MACOS
+        audqt::MenuCommand ({N_("About Audacious"), "help-about"}, aud_ui_show_about_window),
+        audqt::MenuCommand ({N_("Preferences..."), "preferences-system", "Ctrl+,"}, aud_ui_show_prefs_window),
```

Comment:
What's the point of changing the names here?

Reply:
They will show up in the appropriate places in the Mac menu that way.